Dockerfile

```diff
@@ -1,6 +1,8 @@
 # This Dockerfile contains multiple targets.
 # Use 'docker build --target=<name> .' to build one.
 
+ARG ALPINE_VERSION=3.16.2
+
 # devbuild compiles the binary
 # -----------------------------------
 FROM docker.mirror.hashicorp.services/golang:1.17.6 AS devbuild
```

Comment:
Seems like this golang version should be changed too.

Reply:
Actually, I think this is why the tests are failing; settings this to 1.18.5 passes `make e2e-container` for me.